python-lib/language_support.py

```diff
@@ -84,3 +84,47 @@
 
 Models with Creative Commons licenses are not included because this plugin is licensed under Apache-2
 """
+
+SPACY_LANGUAGE_MODELS_LEMMATIZATION = ["en", "es", "nb", "fr", "de", "ru", "pl"]
+
+"""list: Languages that have a SpaCy pre-trained model with a Lemmatizer component"""
+
+SPACY_LANGUAGE_MODELS_MORPHOLOGIZER = ["es", "nb", "fr", "de", "ru", "pl"]
+
+"""list: Languages that have a SpaCy pre-trained model with a Morphologizer component"""
+
+SPACY_LANGUAGE_LOOKUP = [
+    "ca",
+    "cs",
+    "da",
+    "hr",
+    "hu",
+    "id",
+    "it",
+    "lb",
+    "lt",
+    "pt",
+    "ro",
+    "sr",
+    "tl",
+    "tr",
+    "ur",
+    "en",
+    "de",
+    "es",
+    "nb",
+    "fr",
+    "mk",
+    "nl",
+    "sv",
+]
+
+"""list: Languages that have available SpaCy lookup tables for lemmatization. 
+
+The lookup tables are available at https://github.com/explosion/spacy-lookups-data/tree/master/spacy_lookups_data/data """
+
+SPACY_LANGUAGE_RULES = ["bn", "el", "fa"]
+
```

Comment:
For the docstring to be recognized as linked to a variable, it must be directly below with no newline

Reply:
I did it here 09f5159